app/jobs/scheduled/clean_up_inactive_users.rb

```diff
@@ -0,0 +1,29 @@
+module Jobs
+
+  class CleanUpInactiveUsers < Jobs::Scheduled
+    every 1.day
+
+    def execute(args)
+      if SiteSetting.inactive_time_before_cleaning > 0
```

Comment:
We usually do

``` ruby
return if SiteSetting.inactive_time_before_cleaning <= 0
```

That way, we don't have to indent the whole code inside an `if`